dashboard/app/models/unit_group.rb

```diff
@@ -530,15 +530,18 @@ def redirect_to_course_url(user)
   # @param user [User]
   # @return [Boolean] Whether the user can view the course.
   def can_view_version?(user = nil)
+    # Must be part of instructor or participant audience to view version
+    return false unless can_be_participant?(user) || can_be_instructor?(user)
+
```

Comment:
I'm a little fuzzy on when and where we use CanCan.  Would this be a good place to use CanCan?  (Similar comment for `script`).

Reply:
Updated!